### DISCUSSION TITLE
Encrypt resources side client

### DISCUSSION THREAD

**Discussion by vbx** [vmware-tanzu/velero#8163] [General discussions]
Hello, I would like to encrypt velero backup resources but client side. Current, velero backup on minio, but when I download archive tar.gz, and extract it, secret is in plain text. Is there a feature in the future to encrypt client side !?
Thanks

**Comment by kaovilai** [comment]
Had been requested before. Please search velero issues to reduce duplicate requests.
https://github.com/vmware-tanzu/velero/issues/434
https://github.com/vmware-tanzu/velero/issues/3218
